e2e/utils/waits-utils.ts

```diff
@@ -24,7 +24,7 @@ export async function waitForUrl(page: Page, urlSubstr: string): Promise<boolean
     }, {}, urlSubstr);
     return (await jsHandle.jsonValue()) as boolean;
   } catch (e) {
-    console.error(`Wait for URL contains "${urlSubstr}" failed. ${e}`);
+    console.error(`waitForUrl contains "${urlSubstr}" failed. ${e}`);
```

Comment:
`wait-utils.` file changes is not related to the bug. I can undo this file if you would like.